### PR TITLE
Deprecate SbOnce

### PR DESCRIPTION
**Pull request by y4vor**
b/302335657

Test-On-Device: true

Change-Id: Ib6447e3342dc7b2248c6bba219d0ce97b3cc8933

### CODE REVIEW

File: starboard/nplb/extern_c_test.cc [hunk 1]
```diff
@@ -35,6 +35,7 @@ extern "C" {
 #if SB_API_VERSION < 16
 #include "starboard/image.h"
 #endif  // SB_API_VERSION < 16
+#include "starboard/common/once.h"
```
[the remark this answers is not in the document]
Comment (y4vor) [reply starboard/nplb/extern_c_test.cc]: cleaned

File: starboard/nplb/include_all.c [hunk 2]
```diff
@@ -42,7 +42,7 @@
 #include "starboard/memory_reporter.h"
 #include "starboard/microphone.h"
 #include "starboard/mutex.h"
-#include "starboard/once.h"
+#include "starboard/common/once.h"
```
[the remark this answers is not in the document]
Comment (y4vor) [reply starboard/nplb/include_all.c]: cleaned

File: cobalt/base/startup_timer.cc [hunk 3]
```diff
@@ -14,7 +14,7 @@
 
 #include "cobalt/base/startup_timer.h"
 
-#include "starboard/once.h"
+#include "starboard/common/once.h"
```
Comment (andrewsavage1) [inline cobalt/base/startup_timer.cc:17]: If you're going to unconditionally include starboard/common/once.h doesn't that need to include starboard/once.h?

Comment (y4vor) [reply cobalt/base/startup_timer.cc:17]: No, the starboard/once.h is gone. None of the code needs it.

**Pull request merged**